packages/cli/CHANGELOG.md

```diff
@@ -2,6 +2,10 @@
 
 All notable changes to the Zowe CLI package will be documented in this file.
 
+## Recent Changes
+
+- BugFix: Solved daemon issue where Windows usernames were treated as case-sensitive when checking the daemon process owner during Zowe commands.
```

Comment:
I know it may be redundant, but whenever there isn't an issue to link in the changelog, we should at least link the PR.
That way it's easier for the docs-squad to find this PR and read its description to understand the use-case. 😋